src/objective-c/GRPCClient/GRPCCallOptions.h

```diff
@@ -232,6 +240,14 @@ typedef NS_ENUM(NSUInteger, GRPCTransportType) {
  */
 @property(readwrite) NSTimeInterval timeout;
 
+/**
+ * Enable flow control of a gRPC call. The option is default to NO. If set to YES, writeData: method
+ * should only be called at most once before a didWriteData callback is issued, and
```

Comment:
We have the following text in https://github.com/grpc/proposal/pull/136, which contradicts this comment. I think we should update this comment since the API does allow multiple invocations of `didWriteData` if the API caller knows what they are doing :)
```
However, if `writeData` are called multiple times, e.g. due to a race to write messages, gRPC will still
send all of the writes in order and issue `didWriteData` for each invoke of
`writeData`. However in this case users assume their own responsibility of
memory issue, e.g. it may be best to keep track of the number of pending
`writeData` calls without `didWriteData` callback.
```